op-batcher/readme.md

```diff
@@ -0,0 +1,69 @@
+# op-batcher
+
+The `op-batcher` is responsible for ensuring data availability. See the [specs](https://specs.optimism.io/protocol/batcher.html).
+
+
+## Interactions & Dependencies
+The `op-batcher` works together with the [sequencer](../op-node/) (which it reads unsafe blocks from), the data availability layer (e.g. Layer 1 or an [Alt DA](../op-alt-da/) layer, which it posts data to), and the [derivation pipeline](../op-node/) (which reads the data from the DA layer and progresses the safe chain).
+
+It depends directly on some code shared with the derivation pipeline, namely the [`ChannelOut`](../op-node/rollup/derive/channel_out.go) implementation(s). It also depends directly on the shared [txmgr](../op-service/txmgr/) module.
+
+## Testing
```

Comment:
There's also a bunch of op-e2e tests that test some aspects of the batcher.

Reply:
[869b745](https://github.com/ethereum-optimism/optimism/pull/12390/commits/869b745adec5078c53ffab69f047a75abdf9b7a7)